test/e2e/framework/framework.go

```diff
@@ -78,7 +78,7 @@ type Framework struct {
 	SchedulingClient      *schedulingv1client.SchedulingV1Client
 	APIExtensionsClient   *apiextensionsclient.Clientset
 	VPAClient             *vpaclient.AutoscalingV1Client
-	kubeConfigPath        string
+	KubeConfigPath        string
```

Comment:
I assume the diff may be some trailing space over here?

Reply:
The `k` was capitalized actually :)